docs/scos/user/back-office-user-guides/202200.0/merchandising/discount/creating-vouchers.md

```diff
@@ -28,18 +28,18 @@ Review the reference information before you start, or just look up the necessary
 ## Creating a voucher
 
 To create a discount voucher:
-1. On the *Discount* page,  in the top-right corner, click **Create new discount**.
-2. On the *Create new discount* page, in the *General* tab, do the following:
-    1. In *STORE RELATION*, select the stores where you wish the discount to be active.
-    2. In the *DISCOUNT TYPE* drop-down, select **Voucher codes**.
+1. On the **Discount** page,  in the top-right corner, click **Create new discount**.
+2. On the **Create new discount** page, on the **General Information** tab, do the following:
+    1. In **STORE RELATION**, select the stores where you wish the discount to be active.
+    2. In the **DISCOUNT TYPE** drop-down, select **Voucher codes**.
     3. In the **NAME** field, specify the name of the voucher.
-    4. _Optional_:in the **DESCRIPTION** field, enter the description of the voucher.
-    5. _Optional_: in the **PRIORITY** field, enter an integer value from 1 to 9999 for the discount voucher priority. For reference information, see [General information tab](#general-information-tab).
+    4. _Optional_: in the **DESCRIPTION** field, enter the description of the voucher.
+    5. _Optional_: in the **PRIORITY** field, enter an integer value from `1` to `9999` for the discount voucher priority. For reference information, see [General Information tab](#general-information-tab).
     6. Specify if the voucher is exclusive. For reference information, see [General information tab](#general-information-tab).
     7. Specify the validity interval (lifetime) of the voucher.
- 1. Click **Next** or select the **Discount calculation** tab to proceed.
- 2. In the *Discount calculation* tab, do the following:
-    1.  In the **CALCULATOR TYPE** drop-down, select either *Percentage* or *Fixed amount*. For reference information, see [Discount calculation tab](#discount-calculation-tab).
+3. To proceed, click **Next** or select the **Discount calculation** tab.
+4. On the **Discount calculation** tab, do the following:
```

Comment:
```suggestion
4. In the **Discount calculation** tab, do the following:
```

Reply:
According to https://developers.google.com/style/ui-elements, the _on_ preposition should be used here.